tests/mail/tests.py

```diff
@@ -384,6 +400,23 @@ def test_to_header(self):
             email.to, ["list-subscriber@example.com", "list-subscriber2@example.com"]
         )
 
+        email = EmailMessage(
+            subject="test subject",
+            body="test body",
+            from_email="from@example.com",
+            to=["guy1@example.com"],
+            headers={
+                "To": ", ".join(
+                    ["guy1@example.com", "guy2@example.com", "guy3@example.com"]
+                ),
+            },
+        )
+        message = email.message()
+        self.assertEqual(
+            message["To"], "guy1@example.com, guy2@example.com, guy3@example.com"
+        )
```

Comment:
This assertion could be moved to the `test_to_not_repeated` without repeating everything else.